### ISSUE TITLE
Could Anybody help me? I want  to get real data

### ISSUE THREAD

**Issue by DRSEUNGILLEE** [icza/screp#14]
Hello.

Now i just started to analysis broodwar replays using screw 
and I can analyze replay data

![image](https://user-images.githubusercontent.com/43365171/47612002-1ec11300-daac-11e8-9bf7-b17e86589179.png)

but there are a lot of duplications such as production of probe at first time of game,
because players click to make prove even though they don't have enough money.

I want to get real game data not a player's click record. 
Can you guys help me?? how can i get real data ?

![image](https://user-images.githubusercontent.com/43365171/47612004-28e31180-daac-11e8-9bf4-02183dcc7330.png)

**Comment by icza** [comment]
Replays hold user actions. Unfortunately replays do not store if actions are carried out successfully.

For example you may issue a Probe to build a Pylon far away. Replays do not record if the Probe gets to the destination successfully and you have enough minerals to carry out the command.

Unfortunately there is no easy way to tell if a command succeeds. You need the actual game engine and compute everything to know for certain.

So to sum it up, it's impossible to tell for certainty without the game engine.

**Comment by DRSEUNGILLEE** [comment]
> Replays hold user actions. Unfortunately replays do not store if actions are carried out successfully.
> 
> For example you may issue a Probe to build a Pylon far away. Replays do not record if the Probe gets to the destination successfully and you have enough minerals to carry out the command.
> 
> Unfortunately there is no easy way to tell if a command succeeds. You need the actual game engine and compute everything to know for certain.
> 
> So to sum it up, it's impossible to tell with certainty without the game engine.

Thank you very much
